tests/operators/test_google_api_to_s3_transfer.py

```diff
@@ -29,7 +30,7 @@
 class TestGoogleApiToS3Transfer(unittest.TestCase):
 
     def setUp(self):
-        configuration.load_test_config()
+        load_test_config()
```

Comment:
Remove here too.

Reply:
Same here. It's fixed in the merged commit.